code/modules/materials/materials_vr.dm

```diff
@@ -1,4 +1,5 @@
 /datum/material/flesh
```

Comment:
I would kindly ask you to move these /datum/s to materials.dm in compiance with our "Get rid of _v files" policy